index.html

```diff
@@ -4,10 +4,165 @@
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
-  <title>Document</title>
+  <link rel="stylesheet" href="./assets/css/bootstrap-mod.css">
+  <title>Home</title>
 </head>
 <body>
-  
+  <header class="p-3 bg-hero" id="navbar">
+    <nav class="navbar navbar-expand-lg navbar-light p-3 mx-lg-5">
+      <div class="container-fluid">
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarScroll" aria-controls="navbarScroll" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse justify-content-lg-between" id="navbarScroll">
+          <div class="navbar-brand">
+            <img class="generic-logo" src="./assets/img/generic-logo.svg" alt="">
+          </div>
+          <ul class="navbar-nav my-2 my-lg-0 navbar-nav-scroll" style="--bs-scroll-height: 100px;">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="./index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./about.html">About</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="px-3 pt-3 bg-hero pb-5 d-lg-flex flex-lg-column align-items-lg-center justify-content-lg-center">
+      <div class="dk-width p-lg-5 my-lg-5">
+        <h3 class="fw-light text-primary">This year is big! Join us</h3>
+        <h1 class="display-3 mt-2 fw-bold text-primary">Generic Conference Global Nonsens 2023</h1>
+        <p class="dk-text mt-3 mb-4 bg-light p-2 border border-light rounded border-3">Welcome to the annual "Nonsens Conference"! This conference is dedicated to exploring the absurd, the ridiculous, and the downright nonsensical aspects of life. We can't wait to see what absurd ideas and creations you'll bring to the table!</p>
+        <h4 class="fw-bold">2023.12.12 (Thu) - 16 (Sun)</h4>
+        <h6>@Genric Place 234, Otheer Stop and more</h6>
+      </div>
+    </section>
+    <section class="bg-dark bg-pattern-1">
+      <div class="bg-mod d-sm-flex flex-sm-column align-items-sm-center justify-content-sm-center">
+        <div class="p-4 text-white dk-width my-sm-5">
+          <div class="d-flex flex-column align-items-center justify-content-center">
+            <h2>Main program</h2>
+            <span class="spacer"></span>
+          </div>
+          <ul class="list-unstyled d-sm-flex">
+            <li class="container gx-2 p-3 my-4 mx-sm-2 bg-lightgray dk-gallery">
+              <div class="row align-items-center justify-content-center d-flex flex-sm-column text-sm-center">
+                <img class="col-2 col-sm-12" src="./assets/img/award.svg" alt="">
```

Comment:
- [ ] Please, you may need to add `alt `attributes to all your `img `elements; the `alt` attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader). ℹ️